classes/TaskStatusAction.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Класс описывает все состояния задания
+ * и все возможные действия с ним
+ */
+class TaskStatusAction
+{
+    private int $clientId;
+    private int $doerId;
+    public string $status;
+    //public string $action;
```

Comment:
```suggestion
    private string $status;
```
Незачем делать статус публичным